src/js/core/widget/core/Appbar.js

```diff
@@ -351,6 +368,51 @@
 				expandedTitle.style.opacity = expandLevel;
 			};
 
+			/**
+			 * Expand AppBar
+			 * @method expand
+			 * @member ns.widget.core.Appbar
```

Comment:
@public

Reply:
@grzegorz-czajkowski @lslachciak 
We didn't use `@public` (BaseWidget.js) but I see that during of project's live `@public` tag began to appear in source code (wearable/Selector.js). I don't know whether to keep this rule or not

Reply:
I think we should use '@public' tag. This will be a good way to show which API can be used by developers